cmd/binlogctl/main.go

```diff
@@ -59,6 +60,8 @@ func main() {
 		err = ctl.ApplyAction(cfg.EtcdURLs, node.PumpNode, cfg.NodeID, close)
 	case ctl.OfflineDrainer:
 		err = ctl.ApplyAction(cfg.EtcdURLs, node.DrainerNode, cfg.NodeID, close)
+	default:
+		err = errors.Errorf("cmd %s not supported", cfg.Command)
```

Comment:
```suggestion
		err = errors.NotSupportedf("cmd %s", cfg.Command)
```
use this method is simpler